app/services/save_and_invite_provider_user.rb

```diff
@@ -17,7 +17,7 @@ def call
         invite_service.call! if new_user
       end
       invite_service.notify
-    rescue DfeSignInAPIError => e
+    rescue StandardError => e
```

Comment:
What's the intention of relaxing this here? (Specifying `DfeSignInAPIError` was intended to catch the specific case where we couldn't do anything about the result)

Reply:
We broadened it to catch the other error we were experiencing - but I'm happy to change it to keep the DSI error separated

Reply:
hmm, I'm in two minds regarding this, but I think its just a more graceful way to handle those for the provider user if anything were to happen in the wizard, and we won't lose the error as we are sending it to Sentry anyway

Reply:
Yeah it feels to me that this improves user experience in the case where we have a bug, or something unexpected happens. We don't lose out on the sentry error, so we couldn't see any downside

Reply:
If our code is throwing exceptions, the user sees a 500 page and we notify Sentry. This is fine elsewhere. Why not here? If our code is at fault then retrying is unlikely to do anything. If DSI is at fault then retrying is worth a shot, as it can be flaky. So I don't think rescuing a generic error here improves UX.

Reply:
Reverted back to how it was before!